macos/library/FLEKeyEventPlugin.m

```diff
@@ -0,0 +1,57 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "FLEKeyEventPlugin.h"
+
+#import <objc/message.h>
+
+#import "FLEViewController.h"
+
+static NSString *const kKeyEventChannel = @"flutter/keyevent";
+
+@implementation FLEKeyEventPlugin
+
+@synthesize controller = _controller;
+
+- (NSString *)channel {
+    return kKeyEventChannel;
+}
+
+- (void)handleMethodCall:(FLEMethodCall *)call result:(FLEMethodResult)result {
+    NSString *method = call.methodName;
+    NSLog(@"Unhandled text input method '%@'", method);
+}
+
+- (void)dispatchKeyEvent:(NSString *)type event:(NSEvent *)event {
+    // TODO: Do not use 'android'
```

Comment:
I added the issue link to the TODO comment.

Using "fuchsia" instead of android would change the class of the `data` property of the KeyEvent. If we change this handlers in the dart code would need to be adapted. Currently the handler in the example expects `RawKeyEventDataAndroid`. Also I am not filling all expected properties of the data class.
The [Fuchsia](https://github.com/flutter/flutter/blob/09276bea258737e11b109e227f70eac94a4e1691/packages/flutter/lib/src/services/raw_keyboard.dart#L83) implementation has no `keyCode` field, but instead `hidUsage`, not sure how to convert this.

Reply:
> Also I am not filling all expected properties of the data class.

Is there a reason you're not passing modifier key state via metaState? That seems like it would be necessary data for many uses.

Reply:
This wasn't resolved, at it still seems like passing modifier keys will be important, but I'm okay with leaving that for follow-up work.